### DISCUSSION TITLE
Symfony Upgrade from 6.0 to 6.3

### DISCUSSION THREAD

**Discussion by On5-Repos** [symfony/symfony#52121] [Q&A]
I am able to successfully upgrade from 6.0 to 6.3, everything goes smoothly but when I try to browse to the login page it takes around 15 seconds to load. 

I start to experience slowness across all routes? unable to see anything unusual in profiler as upgrade works just fine.

Anyone has any thoughts?

**Comment by ghost** [comment]
No thoughts. Look at the profiler's performance page.

**Reply by On5-Repos** [reply]
<img width="986" alt="image" src="https://github.com/symfony/symfony/assets/46661426/ab508123-5a86-43e4-9a74-cf7199bc7cd7">

Thats all I can see.

**Reply by ghost** [reply]
change threshold to 0

**Reply by On5-Repos** [reply]
with 0 threshold, I can most time consumed by Doctrine / controller

doctrine 3222.6 ms / 6 MiB
controller 5412.4 ms / 6 MiB

**Reply by ghost** [reply]
add screenshot

**Reply by On5-Repos** [reply]
this is performance 
<img width="985" alt="image" src="https://github.com/symfony/symfony/assets/46661426/26c938f8-0057-4ad0-8b2f-fcff991979d3">

<img width="997" alt="image" src="https://github.com/symfony/symfony/assets/46661426/2f014b35-c013-415b-b7c2-35a71611ab45">

**Reply by ghost** [reply]
I would start with Doctrine. It has it's own tab in the profiler. You can check how long queries take.

**Reply by On5-Repos** [reply]
same query is called like 26 times

**Reply by ghost** [reply]
Debug it. Check who calls this query so many times.
